src/System/DataLogger.cpp

```diff
@@ -68,7 +67,7 @@ bool DataLogger::open(String name, int numColumns)
 
         DEBUG_SERIAL_LN("Loading " + name);
 
-        File _curFile = SD.open(name, FILE_WRITE | FILE_READ);
+        File _curFile = SD.open(name, FILE_WRITE);
```

Comment:
Can we read from a file if we only specify the FILE_WRITE mode?

Reply:
I changed that because the definition of ```FILE_WRITE``` in the SD card library is:

```
#define FILE_WRITE (O_READ | O_WRITE | O_CREAT | O_APPEND)
```

So adding ```FILE_WRITE | FILE_READ``` is redundant

Reply:
Oh :0 I see